tsdb/engine/tsm1/data_file.go

```diff
@@ -185,6 +221,33 @@ func (d *directIndex) Entry(key string, t time.Time) *IndexEntry {
 	return nil
 }
 
+func (d *directIndex) Contains(key string) bool {
```

Comment:
Not sure I follow why this is true. The key might be greater lexicographically that then the max key in the index, no?